code-challenges.js

```diff
@@ -9,36 +9,63 @@
 // --------------------INSTRUCTOR EXAMPLE: Create a function that evaluates two strings and determines which of the strings has more characters. Use the two sets of test variables provided.
 
 // Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
-// Expected outcome: "banana"
+// const fruit1 = "apple"
+// const fruit2 = "banana"
+// // Expected outcome: "banana"
 
-// Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
-// Expected outcome: "cherry"
+// // Set two:
+// const fruit3 = "cherry"
+// const fruit4 = "kiwi"
+// // Expected outcome: "cherry"
 
 // Pseudo code:
 
 
 // --------------------1) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-const padres1998WorldSeriesRuns = [6, 3, 5, 3]
-// Expected output: 9
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// // // Expected output: 9
 
-// Pseudo code:
-
-// --------------------2) Create the code that will reverse the letters of a string using the test variable provided below.
-
-const currentCohort = "LEARN 2023"
-// Expected output: "3202 NRAEL"
+// // Pseudo code: Input needs to be the two arrays, and I will need to concatenate them to create a new array
+//                 //Output will be the size of the array, not the sum.
 
-// Pseudo code:
-
-// --------------------3) Use a for loop to log only the odd numbers from the array using the test variable provided below.
+//  const numberOfWorldSeries = (array1, array2) => {
+//         let newArray = array1.concat(array2)
+//         return newArray
 
-const stockExchange = [13, 34, -4, 42, 5, -5, 10, 27, 42, 10]
-// Expected output: 13 5 -5 27
+//  }               
+// console.log(numberOfWorldSeries(padres1984WorldSeriesRuns, padres1998WorldSeriesRuns).length)
```

Comment:
I would encourage you to make sure all your logic is contained inside the function.